.github/workflows/lint.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
I think we'll want to update the [python version in test.yaml](https://github.com/uwcirg/carl/blob/develop/.github/workflows/test.yaml#L9) too

That can wait, and I may have missed that you're keeping 3.7 until this is merged